ruby/ruby_releases/ruby_benchmarks/runner

```diff
@@ -45,3 +45,14 @@ if [ "$OPTCARROT_BENCHMARK" = true ]; then
 else
   echo "Skipping optcarrot benchmarks"
 fi
+
+if [ "$LIQUID_BENCHMARK" = true ]; then
+  echo "Running liquid benchmarks with Ruby $RUBY_VERSION"
+  gem update bundler
+  gem install bundler --no-ri --no-rdoc
```

Comment:
Why `gem update` and then `gem install`? `gem install bundler --no-ri --no-rdoc` should be enough (kudos for using --no-ri --no-rdoc)